libraries/core_libs/storage/include/storage/storage.hpp

```diff
@@ -111,8 +111,9 @@ class DbStorage : public std::enable_shared_from_this<DbStorage> {
     COLUMN(pbft_cert_voted_block);
     COLUMN(pbft_head);
     COLUMN(verified_votes);
-    COLUMN(soft_votes);  // only for current PBFT round
-    COLUMN(next_votes);  // only for previous PBFT round
+    COLUMN(soft_votes);             // only for current PBFT round
+    COLUMN(next_votes);             // only for previous PBFT round
+    COLUMN(last_block_cert_votes);  // cert votes for last block in pbft chain
```

Comment:
isnt this already stored in db ?

Reply:
the whole vote handling is kind of complex and probably needs refactoring. I know that the same cert votes I am storing in last_block_cert_votes were probably already stored in one of these *_votes tables but I found it too complex to reuse it for now

Reply:
I think that as for just storing votes it could be ok as is, I think having only cert votes for the previous block in a separate column so that they can easily be fetched in a batch when required by syncing or finalizing next block is better than mixing these cert votes with cert votes for the following block and then having to filter them out when pulling them. This votes move from one column to the other is quite fast and it is only done once on block finalization. I mean it is possible that some other design could be more efficient and whole vote handling needs a refactor but I am not sure how to put that in a simple comment